### ISSUE TITLE
A documentation example for C++ API

### ISSUE THREAD

**Issue by hombit** [exoplanet-dev/celerite2#52]
Please, can you add a C++ example to the documentation? `celerite` C++ doc's example looks helpful but I'm not sure if it is applicable here. Thank you!

**Comment by dfm** [comment]
Yeah - that would be useful and the interface is very different from `celerite`. I'm not sure when I'll be able to get to this, but some combination of the [API docs](https://celerite2.readthedocs.io/en/latest/api/c++/) and the [C++ unit tests](https://github.com/exoplanet-dev/celerite2/tree/main/c%2B%2B/test) might get you started?
